src/chain/network_event.rs

```diff
@@ -69,7 +69,7 @@ impl SectionInfoSigPayload {
         info: &SectionInfo,
         full_id: &FullId,
     ) -> Result<SectionInfoSigPayload, RoutingError> {
-        let proof = Proof::new(*full_id.public_id(), full_id.signing_private_key(), &info)?;
+        let proof = Proof::new(full_id, &info)?;
```

Comment:
Should that not have been part of a previous commit?

Reply:
It looks like that. Not sure how it happened. Do you want me to do something about it?

Reply:
Could move it to the right commit, but I'll let you decide if it is worth it.